vendor/cmark/cmark.odin

```diff
@@ -0,0 +1,464 @@
+/*
+	Bindings against CMark (https://github.com/commonmark/cmark)
+
+	Original authors: John MacFarlane, Vicent Marti, Kārlis Gaņģis, Nick Wellnhofer.
+	See LICENSE for license details.
+*/
+package cmark
+
+import "core:c"
+import "core:c/libc"
+
+BINDING_VERSION :: Version_Info{major = 0, minor = 30, patch = 2}
+
+when ODIN_OS == .Windows {
+	// To use `import "vendor:cmark"` and copy the `.dll` next to your program.
```

Comment:
Any reason it has to be a DLL and cannot be statically linked?

Reply:
I'll have a look at the nmake file to see it it can be built statically.